packages/gatsby-theme-docs/src/components/seo.js

```diff
@@ -61,6 +61,10 @@ const SEO = (props) => {
       name: 'twitter:description',
       content: metaDescription,
     },
+    {
+      name: 'commercetools:title-for-onsite-search',
+      content: `${siteContextTitle} > ${siteData.siteMetadata.title}`,
```

Comment:
You need to cover the case when the current microsite does not have a site context title available.  In that case just stick to the title alone.